keyboards/m0lly/README.md

```diff
@@ -0,0 +1,17 @@
+The Key Company M0LLY
+
+[TKC M0LLY](https://cdn.shopify.com/s/files/1/1679/2319/files/Molly_Terminal_1024x1024.jpg?v=1529067702)
+
+
+The Key Company M0LLY is a keyboard inspired by the Apple Macintosh M0110A utilizing the AT90USB1286 microcontroller.
```

Comment:
This feels like awkward phrasing. 

```suggestion
The Key Company's M0LLY keyboard is a keyboard inspired by the Apple Macintosh M0110A utilizing the AT90USB1286 microcontroller.
```

Reply:
ssd1306.h in /drivers/avr expects i2c.h not i2c_master.h. I know there's currently a restructure going on with ssd1306.h. Do I need to wait for that to be completed before I can get this merged?

Switching to <i2c_master.h> causes compile to fail at drivers/avr/ssd1306.c:

`Compiling: drivers/avr/ssd1306.c                                                                   drivers/avr/ssd1306.c:4:17: fatal error: i2c.h: No such file or directory
 #include "i2c.h"
                 ^
compilation terminated.
 [ERRORS]`